### PR TITLE
Catch some errors and remove unused method

### PR DESCRIPTION
**Pull request by dkanada**

### CODE REVIEW

File: src/controllers/streamingsettings.js [hunk 1]
```diff
@@ -30,24 +31,6 @@ define(["jQuery", "libraryMenu", "loading"], function ($, libraryMenu, loading)
     }
 
     $(document).on("pageinit", "#streamingSettingsPage", function () {
-        var page = this;
-        $("#btnSelectTranscodingTempPath", page).on("click.selectDirectory", function () {
```
Comment (JustAMan) [inline src/controllers/streamingsettings.js]: why is this no longer used? is transcoding path no longer controllable?

File: src/controllers/dashboard/general.js [hunk 2]
```diff
@@ -56,6 +56,12 @@ define(["jQuery", "loading", "fnchecked", "emby-checkbox", "emby-textarea", "emb
                         }
                     });
                 });
+            }, function () {
```
Comment (JustAMan) [inline src/controllers/dashboard/general.js]: don't we have to make this being called in `.then()` of inner calls, too? right now it only fires on error of `.updateServerConfiguration()` if I understand correctly

File: src/controllers/streamingsettings.js (hunk 1 above)
Comment (dkanada) [reply src/controllers/streamingsettings.js]: It's controllable in a separate page, this must have been left over from some other time.

File: src/controllers/dashboard/general.js (hunk 2 above)
Comment (dkanada) [reply src/controllers/dashboard/general.js]: Technically yes, but I was only adding this in response to a specific error related to the server configuration, so I didn't want to track down every other API call and add an error handler.

Comment (JustAMan) [reply src/controllers/dashboard/general.js]: So looks like we need a ticket for tracking that thing... could fix later.
Do we have a ticket for your "specific error"?

Comment (dkanada) [reply src/controllers/dashboard/general.js]: No ticket, I just found and fixed it in like fifteen minutes.

**Pull request merged**